internal/common/utils.go

```diff
@@ -241,10 +240,11 @@ func MakeAddressable(name string, addr *models.Addressable) (*models.Addressable
 }
 
 func VerifyIdFormat(id string, objName string) error {
-	if len(id) != 24 || !bson.IsObjectIdHex(id) {
-		errMsg := fmt.Sprintf("Add %s returned invalid Id: %s", objName, id)
-		LoggingClient.Error(errMsg)
-		return fmt.Errorf(errMsg)
-	}
+	//this function will be re-write after all the id are replaced to UUID
+	//if len(id) != 24 || !bson.IsObjectIdHex(id) {
+	//	errMsg := fmt.Sprintf("Add %s returned invalid Id: %s", objName, id)
+	//	LoggingClient.Error(errMsg)
+	//	return fmt.Errorf(errMsg)
+	//}
```

Comment:
Do we want to consider removing this function entirely? Since the Id is now a string, it could literally be anything. Sure, we're moving toward UUIDs but that's a utilitarian choice for generating unique IDs. Do we really need to be strict about the Id's format?

Reply:
How about just checking whether it is empty string for now and leave this function for the future

Reply:
I'm good with that.